src/clj/harja/palvelin/palvelut/laadunseuranta/talvihoitoreitit_palvelu.clj

```diff
@@ -16,7 +16,8 @@
 (defn hae-urakan-talvihoitoreitit [db user {:keys [urakka-id]}]
   (log/debug "hae-urakan-talvihoitoreitit ::user" user)
   ;; Estä muut, kuin järjestelmävastaavat näkemästä talvihoitoreittejä
-  (when (contains? (:roolit user) "Jarjestelmavastaava")
+  (when (or (contains? (:roolit user) "Jarjestelmavastaava")
```

Comment:
Olisko setti jo nätimpi? 
(#{"Jarjestelmavastaava" "ELY_Urakanvalvoja} (:roolit user))

Reply:
Olisi 👍  Mutta en muuta, kun tämä muuttuu kuitenkin, kunhan saadaan oikeat käyttöoikeudet käyttöön.

Reply:
Ei itseasiassa pystykään tekemään näin. Tuo ELY_Urakanvalvoja on tarkistaa urakkakohtaisena roolina. Päädyin muuttamaan vähän tarkistusta.